.circleci/config.yml

```diff
@@ -86,8 +86,7 @@ jobs:
           command: make docs
       - run:
           name: "MkDocs"
-          # TODO(spencer): Update with new url
-          command: MK_DOCS_SITE_URL="https://head.coda.io/developers/temp-developer-docs/packs/latest/" make build-mkdocs
+          command: MK_DOCS_SITE_URL="https://head.coda.io/packs/build/packs/latest/" make build-mkdocs
```

Comment:
yeah it's a bit confusing. `https://head.coda.io/packs/build/` appropriately redirects when you hit it in the browser, but the extra `packs` one is the "source of truth" URL that appropriately routes to our S3 bucket (it's a bit weird here because before the plan was a top-level `developers` and so the folder in S3 is named `packs`)

Reply:
Is it possible to fix that? If the version with the extra `/packs` is marked as canonical then Google will likely end up sending traffic there.

Reply:
oh hmm i see ok let me look into this more

Reply:
okay going to try without the extra one and test it out